applications/main/subghz/subghz_cli.c

```diff
@@ -1067,7 +1067,7 @@ static void subghz_cli_command_chat(Cli* cli, FuriString* args) {
                         furi_string_push_back(output, message[i]);
                         if(message[i] == '\n') {
                             printf("\r");
-                            for(uint8_t i = 0; i < 80; i++) {
+                            for(uint8_t j = 0; j < 80; i++) {
```

Comment:
To be honest this specific place was just shadowing previous declaration without causing any errors, but now `i++` increments completely different thing.

Reply:
@skotopes on the other hand, it still looks like a bug. 
But, if no one has any complaints, then it’s already a feature. 
But still I would add a comment with a link to the current pull request

Reply:
Just fix `i++` thing )

Reply:
@skotopes updated. As I understand this loop just create extra space to put line on the side.